src/http2_connection.erl

```diff
@@ -3,10 +3,8 @@
 -behaviour(gen_fsm).
 
 -include("http2.hrl").
-
+-compile(export_all). %% for now
```

Comment:
oops! I was doing wholesale API changes and wasn't sure how it would end. I'll fix it